gubernator/testgrid.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import cloudstorage as gcs
+
+import pb_glance
+
+
+CONFIG_PROTO_SCHEMA = {
+    1: {
+        'name': 'test_groups',
+        1: 'name',
+        2: 'query',
+        9: {},
+    },
+    2: {
+        'name': 'dashboards',
+        1: {
+            'name': 'dashboard_tab',
+            1: 'name',
+            2: 'test_group_name',
+            7: {},
+            8: {2: {}},
+            9: {},
+            11: {},
+            12: {},
+        },
+        2: 'name',
+    }
+}
+
+_testgrid_config = None
+
+
+def get_config():
+    global _testgrid_config  # pylint: disable=global-statement
```

Comment:
If you need it to be loaded once and then used multiple times then load it once at the start and then pass it to whoever uses it.